doc/sources/conf.py

```diff
@@ -31,6 +31,10 @@
 #
 import os
 import sys
+import json
```

Comment:
lint fails on imports - I think it probably wants json before sys

alternatively can run isort on the file to ensure lint check passes